packages/ajax/src/AjaxClient.js

```diff
@@ -27,11 +27,11 @@ export class AjaxClient {
       xsrfCookieName: 'XSRF-TOKEN',
       xsrfHeaderName: 'X-XSRF-TOKEN',
       jsonPrefix: '',
+      ...config,
```

Comment:
Ah good catch

Reply:
I would see this as a bugfix, if you have time could you add a regression test for this fix? And then we should release it as a patch as well and make clear what was fixed in the changelog, by running `npx run changeset` :)

Reply:
Sure, planning to :)